lang/shared/src/main/scala/com/wavesplatform/lang/v1/evaluator/ctx/impl/waves/Bindings.scala

```diff
@@ -18,9 +18,9 @@ object Bindings {
 
   private def provenTxPart(tx: Proven): Map[String, Any] =
     Map(
-      "sender"    -> tx.sender,
-      "senderPk"  -> tx.senderPk,
-      "bodyBytes" -> tx.bodyBytes,
+      "sender"          -> CaseObj(addressType.typeRef, Map("bytes" -> tx.sender.bytes)),
```

Comment:
This is a duplicate mapping as of MapRecipient, it's best to avoid that.
Another case where this mapping is needed is `order.sender/senderPk`, so let's refactor that to separate function